src/sentry/apidocs/parameters.py

```diff
@@ -0,0 +1,28 @@
+from drf_spectacular.utils import OpenApiParameter
+
+
+class GLOBAL_PARAMS:
```

Comment:
Interesting. I take it these are just here for reusability, and the class is just for convenient semantic grouping, that the framework isn't going to consume the class itself?

I think the name `GlobalParams` would be more in line with the way we format enum classes, e.g. `Outcome.ACCEPTED` or `OrganizationStatus.ACTIVE`.